website/content/docs/platform/k8s/helm/enterprise.mdx

```diff
@@ -46,6 +48,38 @@ Once the cluster is [initialized and unsealed](/docs/platform/k8s/helm/run), you
 kubectl exec -ti vault-0 -- vault license get
 ```
 
+### License Update
+
+To update the autoloaded license in Vault, you may do the following:
+
+- Delete and recreate the secret
```

Comment:
Why suggest delete/recreate as an option, instead of just updating?  I'm a little concerned about the possibility of a restart during the interval where the secret doesn't exist.

Reply:
+1, although it's more complex, let's document the appropriate `kubectl patch` command here.

Reply:
Yeah I debated this a bit and went for simplicity, but I agree patching is better. Changed in d3e7aa8183defd0e013a20774ea3f32a3965550d. @tomhjp let me know if that's what you had in mind.